### PR TITLE
first draft of documentation for JupyterGIS

### PR DESCRIPTION
**Pull request by annefou**
based on JupyterCAD documentation

### CODE REVIEW

File: docs/source/contributing.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.. _contributing:
+
+============
+Contributing
+============
+
+Development install
+-------------------
+
+**Note:** You will need :ref:`Node.js <https://nodejs.org/>` and :ref:`Docker <https://www.docker.com/>` to build the extension package.
```
Comment (martinRenou) [inline docs/source/contributing.rst:10]: Unless I missed something, I believe we don't need Docker to build jupytergis locally

```suggestion
**Note:** You will need :ref:`Node.js <https://nodejs.org/>` to build the extension package.
```

**Review by martinRenou — COMMENTED** [review]
Thanks! I just have a tiny suggestion otherwise it looks good!

I will setup readthedocs properly for the repo so we can see a preview once this is merged.

**Comment by brichet** [comment]
Thanks @gjmooney.
Do we already have a `jupyterGIS` project in readthedocs ?
Should we include a *.readthedocs.yaml* file to update it automatically ?

**Comment by martinRenou** [comment]
I just created the project on readthedocs https://readthedocs.org/projects/jupytergis

**Review by martinRenou — APPROVED** [review]
Thanks! Let's get this in and we can finalize the readthedocs setup separately

**Pull request merged**